libxrdp/xrdp_rdp.c

```diff
@@ -44,6 +45,9 @@ xrdp_rdp_read_config(struct xrdp_client_info *client_info)
     char *item = (char *)NULL;
     char *value = (char *)NULL;
     char cfg_file[256];
+    char *p = (char *)NULL;
+    char *tmp = (char *)NULL;
```

Comment:
I think so. If it's close enough to be visible in the context, perhaps it should be fixed too.